tests/src/main/java/testcases/T068_scope_factory/config.pro

```diff
@@ -0,0 +1,9 @@
+-dontshrink
+-keep class **Test {
+    public static void run();
+}
+
+-keepnames @motif.Scope interface *
+-keepnames @motif.ScopeImpl class * {
```

Comment:
This is slightly bad because we only actually need to keep the class name intact whereas this forces proguard to keep the class name + all names of class members intact 
if you consider applications in the order of 1000s of scopes that is a lot of names and especially bad if proguard applies that to private members too (I hope not but now have to verify)

Reply:
I believe this only keeps the constructor name.

This [sample.zip](https://github.com/uber/motif/files/3699064/out.zip) demonstrates this behavior. (Unzip and run `./run.sh`)

Reply:
Yep lgtm. Weird that the proguard documentation leaves it ambiguous 
For reference:
https://www.guardsquare.com/en/products/proguard/manual/usage

> -keepnames class_specification
Short for -keep,allowshrinkingclass_specification Specifies classes and class members whose names are to be preserved, if they aren't removed in the shrinking phase. For example, you may want to keep all class names of classes that implement theSerializable interface, so that the processed code remains compatible with any originally serialized classes. Classes that aren't used at all can still be removed. Only applicable when obfuscating.